temporal-sdk/src/main/java/io/temporal/internal/sync/SyncWorkflowContext.java

```diff
@@ -732,6 +732,11 @@ public <R> ChildWorkflowOutput<R> executeChildWorkflow(ChildWorkflowInput<R> inp
   @Override
   public <R> ExecuteNexusOperationOutput<R> executeNexusOperation(
       ExecuteNexusOperationInput<R> input) {
+    Preconditions.checkArgument(
+        input.getEndpoint() != null && !input.getEndpoint().equals(""), "endpoint must be set");
```

Comment:
```suggestion
        input.getEndpoint() != null && !input.getEndpoint().isEmpty(), "endpoint must be set");
```

Can use `isEmpty` on these

Reply:
done